### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by M-fazin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ The developer team is a community around the world for supporting developers and
 **The Developer Team Members:**
 
 - [Fayas Noushad](https://github.com/FayasNoushad)
+- [MUHAMMED FAZIN.E.K](https://github.com/M-fazin)
```
Comment (FayasNoushad) [inline README.md:14]: Please add as Capital letters and small letters. `Muhammed Fazin EK`

Comment (M-fazin) [reply README.md:15]: Ok

File: README.md [hunk 2]
```diff
@@ -12,6 +12,8 @@ The developer team is a community around the world for supporting developers and
 
 - [Fayas Noushad](https://github.com/FayasNoushad)
```
Comment (FayasNoushad) [inline README.md:14]: Remove this line. Number 14.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,8 @@ The developer team is a community around the world for supporting developers and
...
 
+- [Muhammed Fazin.E.K](https://github.com/M-fazin)
```
Comment (FayasNoushad) [inline README.md:15]: Change name to `Muhammed Fazin EK`

**Review by FayasNoushad — APPROVED** [review]

**Pull request merged**